.gitignore

```diff
@@ -1,21 +1,6 @@
-# If you prefer the allow list template instead of the deny list, see community template:
-# https://github.com/github/gitignore/blob/main/community/Golang/Go.AllowList.gitignore
-#
-# Binaries for programs and plugins
-*.exe
-*.exe~
-*.dll
-*.so
-*.dylib
-
-# Test binary, built with `go test -c`
-*.test
-
-# Output of the go coverage tool, specifically when used with LiteIDE
-*.out
-
-# Dependency directories (remove the comment below to include it)
-# vendor/
+# build and vscode are not needed
+build/
```

Comment:
Can we keep the previous diff too?

Reply:
You mean keeping the `*.exe|*.test|*.out` stuff as well?

Reply:
Keeping `*.out` for sure. The rest we can live with -- the .gitignore file currently follows a generic template.

Reply:
Ok, I will read it. `exe|test` were not very useful, that's why I removed them ;-)